### DISCUSSION TITLE
Why was Xournal rewritten to Xournal++?

### DISCUSSION THREAD

**Discussion by K4LCIFER** [xournalpp/xournalpp#3770] [General]
The original Xournal program was written in C as far as I'm aware, and Xournal++ is a rewrite of Xournal in C++. What was the reasoning behind why this was decided instead of just continuing development in C? Was there something inherently better with C++ that made it a better choice to work with for designing a desktop application over C?

**Comment by rolandlo** [comment]
See https://github.com/xournalpp/xournalpp/issues/3574
